### PR TITLE
 rule-maintenance - 7 - old - 2

### PR DESCRIPTION
**Pull request by SKEIDs**

### CODE REVIEW

**Comment by Yuki2718** [comment]
Looks generally good, but please divide into smaller PRs next time, otherwise it takes too much time for review.

**Comment by SKEIDs** [comment]
I understand. I will be careful from now om. I apologize for the inconvenience

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -1524,8 +1513,6 @@ autoc-one.jp#$#body.modal-open { overflow: auto!important;padding-right: auto!im
 denverpost.com#$#body { overflow: auto !important; }
 denverpost.com#$#.modal-scrollable { display: none !important; }
 denverpost.com#$#.modal-scrollable ~ div.connext-modal-backdrop { display: none !important; }
-lostiempos.com,nazya.com,firstthings.com###colorbox
-lostiempos.com,nazya.com,firstthings.com###cboxOverlay
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/popups.txt:1528]: I'mt not sure but think it's best to keep rules like these as are. These sites probably belong to the same group so it's convenient to place them side-by-side, in fact I kinda intentionally didn't merge some rules in JPF for similar reason.

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -2892,15 +2868,14 @@ profile.ru#$##subscribeModal { display: none!important; }
...
-msc.com##.reveal-modal-bg
 mangatown.com,mangahere.cc###edShow
 coolmod.com#$#.sweet-overlay { display: none!important; }
 coolmod.com#$#.sweet-alert { display: none!important; }
 coolmod.com#$#body { overflow: visible!important; }
 5lb.ru###rees46-popup
 stroi.mos.ru##.subscribe-popup__wrapper
 iosninja.io###social-modal
-gameloft.com##.fancybox-overlay
+ciapc.com.br,pravmir.ru,jeans-meile.de,sportfm.ru,gameloft.com##.fancybox-overlay
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/popups.txt:2878]: `##fancybox-overlay` and `###fancybox-content` are interrelated, so better to put them side-by-side.

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/popups.txt:1528]: ok

File: AnnoyancesFilter/sections/popups.txt (hunk 2 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/popups.txt:2880]: ok.

**Pull request closed without merging**